pyiron_base/archiving/export.py

```diff
@@ -0,0 +1,63 @@
+import os
+import numpy as np
+from shutil import copyfile
+from pyfileindex import PyFileIndex 
+from pyiron_base import Project
```

Comment:
We have to be a bit careful with the import of the `Project` class as this functionality should be imported in the `Project` class. I would suggest just passing a `Project` object instance and then using `Project().open()`.

Reply:
Nice point. I removed it, and used `Project.open(<path>)`. Thanks for pointing it out.